packages/hs-react-ui/src/components/Label/Label.tsx

```diff
@@ -1,25 +1,83 @@
 import React from 'react';
 import styled from 'styled-components';
-
+import Icon from '@mdi/react';
+import { mdiCheckBold, mdiAsterisk } from '@mdi/js';
 import colors from '../../enums/colors';
 
 export const StyledLabel = styled.label`
-  ${({ color = colors.grayDark }: { color: colors | string }) => `
-    display: block;
+  ${({ color = colors.grayLight }: { color: colors | string }) => `
+    display: inline-flex;
     color: ${color};
-    text-transform: uppercase;
+    font-family: Arial, Helvetica, sans-serif;
     margin-bottom: .25em;
-    font-size: .75em;
+    font-size: .8em;
+  `}
+`;
+
+export const Child = styled.input`
+  ${({ color = colors.grayLight }: { color: colors | string }) => `
+  display: block;  
+  border: solid 2px ${color};
+  width: 15%;
   `}
 `;
 
+export const StyledDiv = styled.div`
+  
+`;
+
+const IconContainer = styled.span`
+  display: inline-flex;
+  margin-left: 0.25rem;
+`;
+
 export interface LabelProps {
   labelText: string;
   color?: colors | string;
+  isValid: boolean;
+  checkValidity: boolean;
+  colorValid: colors | string;
+  colorInvalid: colors | string;
+  htmlFor: string;
+  isRequired: boolean;
 }
 
-const Label = ({ labelText, color = colors.grayDark }: LabelProps) => (
-  <StyledLabel color={color}>{labelText}</StyledLabel>
-);
+const Label = ({
+  labelText,
+  color = colors.grayLight,
+  isValid = false,
+  checkValidity = false,
```

Comment:
This may not be needed if you don't default isValid to false. That way you can just display validation if `isValid !== undefined`
```suggestion
  isValid,
```